### PR TITLE
feat(readme): many things required for assignment

### PR DESCRIPTION
**Pull request by jamesfulford**

### CODE REVIEW

**Comment by jamesfulford** [comment]
Not ready yet, but just opening PR for visibility/early reviews

**Review by alejandrojsuazohvd — APPROVED** [review]
Read through. It looks good.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,95 @@
-# MoneyWise2020
+# Money Wise
```
Comment (sarabjotanand) [inline README.md:1]: Perhaps make it easier for staff to see what it stands for. i.e. **Team Name:** Money Wise

File: README.md [hunk 2]
```diff
@@ -1,6 +1,95 @@
...
```
Comment (sarabjotanand) [inline README.md:2]: Need to add Team Name in Readme aswell. 
Team Name: MoneySages

**Review by rscronce — APPROVED** [review]

**Pull request merged**